### ISSUE TITLE
GP practice level deprivation data returns 0 rows of data

### ISSUE THREAD

**Issue by miqdadasaria** [ropensci/fingertipsR#83]
Hi

I am trying to pull down IMD 2015 scores for all GP practices using the fingertipsR API using the following code:

library("fingertipsR")
practice_imd = deprivation_decile(AreaTypeID=7, Year=2015)

This is returning zero rows of data. It seems to return data for some other AreaTypeIDs such as 102 and 3 but not for AreatTypes of either 7 or 101.

Any idea what is going wrong?

Is there a csv file somewhere that has these IMD data for all GP practices (ideally including those that have closed in recent years) that can be downloaded from somewhere on the PHE website?

Is there a description of the methodology used to attribute IMD scores to GP practices? 

Thanks

**Comment by sebastian-fox** [comment]
Hi @miqdadasaria - thanks for pointing this out. I will make an update in the next couple of days hopefully to the development version of the package which should allow you to pick up your GP deprivation levels. 

I will also find out how these scores are attributed to GP practice. Following the [definitions page](https://fingertips.phe.org.uk/profile/general-practice/data#page/6/gid/2000005/pat/152/par/E38000001/ati/7/are/B83620/iid/93553/age/1/sex/4) it looks like [this site](https://indicesofdeprivation.co.uk/) will have some information. The person who puts these scores into Fingertips is away at the moment, so these links might be useful in the meantime.

**Comment by miqdadasaria** [comment]
Thanks @sebastian-fox looking forward to the update and thanks for the links

**Comment by sebastian-fox** [comment]
Ok - I've pushed some updates to the package that should now work for you. You'll need to install the development version of it:
```
remotes::install_github("rOpenSci/fingertipsR",
                        build_vignettes = TRUE,
                        dependencies = "suggests")
```
As for documentation around the methodology, the earliest I'd be able to get back to you is Monday. If you're happy with the links I've given you already and the updates I've made to the package, can I ask that you close the issue - otherwise, I'll get back to you about methodology when I hear something from my colleague.
Many thanks
